ctapipe/analysis/Sensitivity.py

```diff
@@ -0,0 +1,790 @@
+from itertools import chain
+import astropy.units as u
+from astropy.table import Table
+from scipy.optimize import minimize
+import scipy.integrate as integrate
+
+import numpy as np
+
+from matplotlib import pyplot as plt
+
+
+# tau, the superior circle number
+np.tau = 2*np.pi
+
+__all__ = ["SensitivityPointSource"]
+
+
+def convert_astropy_array(arr, unit=None):
+    """
+    converts a python list of quantities into a quantified numpy array of the same
+    dimension
+
+    Parameters
+    ----------
+    arr : python list
+        list of quantities of same dimension (not strictly exact same unit)
+    unit : astropy unit, optional (default: None)
+        if set, uses this as the unit of the numpy array; if not, uses the unit of the
+        first element
+        ought to be of same dimension of the quantities in the list (there is no test)
+
+    Returns
+    -------
+    a : quantified numpy array
+    """
+
+    if unit is None:
+        unit = arr[0].unit
+    return np.array([a.to(unit).value for a in arr])*unit
+
+
+def crab_source_rate(energy):
```

Comment:
[also concerning the comment to `CR_background_rate`]
Of course, these rate definitions here are not intended to be the standard definitions to be used in ctapipe. I just wanted to have something to get things going. Hence also no proper references other than "random plot I found online". I have no problem referring to `gammapy` instead (or some dedicated `ctapipe` flux module if it is not desired to `import gammapy`)